### ISSUE TITLE
Please check setting 6 data bits in 'open' method. Thanks.

### ISSUE THREAD

**Issue by vladvil** [kpishere/POSIXSerialPort#5]
Shouldn't case 6: use CS6? Here is the current code:

  switch (self.numberOfDataBits) {
        case 5:
            options.c_cflag |= CS5;
            break;
        case 6:
            options.c_cflag |= CS5;
            break;
        case 7: 
...

**Comment by kpishere** [comment]
Yes, thanks.  It is corrected .. .cc9b3671dba89bafb5521b13959cb3d7d6deb033

**Comment by kpishere** [comment]
Yes, thanks.  It is corrected .. .cc9b3671dba89bafb5521b13959cb3d7d6deb033

**Comment by vladvil** [comment]
corrected

**Comment by vladvil** [comment]
closed

**Comment by vladvil** [comment]
closed
